framework/interface/templates/target.html

```diff
@@ -128,6 +143,13 @@ <h4 class="modal-title" id="pluginOutputFilterModalLabel">Advanced Filter</h4>
     }
 }
 
+function modifyMapping(radio, filter_type, data){
+    if (typeof mySpace.filter_data[filter_type] === 'undefined') mySpace.filter_data[filter_type] = [];
```

Comment:
Don't use inline if statements. Better to break it up and indent for easy reviewing.